src/new-components/button-link/button-link.module.css

```diff
@@ -0,0 +1,9 @@
+.container {
+    gap: var(--reactist-spacing-medium);
```

Comment:
It's a grid and a [flex property](https://caniuse.com/flexbox-gap). In flex it is not supported in IE11 (not a problem for Reactist and Twist) but it is not yet supported in Safari, although it will be supported soon (it is already supported in Safari Technology Preview). I acknowledge that this is a problem, but I wanted to worry about that later.